input_topic: "floam_odom"
output_topic: "rtabmap/odom"
child_frame_id: "base_link"